aiida/backends/tests/tcodexporter.py

```diff
@@ -174,35 +175,35 @@ def test_cif_structure_roundtrip(self):
         calc.add_link_from(code, "code")
         calc.set_option('environment_variables', {'PATH': '/dev/null', 'USER': 'unknown'})
 
-        with tempfile.NamedTemporaryFile(mode='w+', prefix="Fe") as f:
-            f.write("<UPF version=\"2.0.1\">\nelement=\"Fe\"\n")
-            f.flush()
-            upf = UpfData(file=f.name)
+        with tempfile.NamedTemporaryFile(mode='w+', prefix="Fe") as tmpf:
+            tmpf.write("<UPF version=\"2.0.1\">\nelement=\"Fe\"\n")
+            tmpf.flush()
+            upf = UpfData(file=tmpf.name)
             upf.store()
             calc.add_link_from(upf, "upf")
 
-        with tempfile.NamedTemporaryFile(mode='w+') as f:
-            f.write("data_test")
-            f.flush()
-            cif = CifData(file=f.name)
+        with tempfile.NamedTemporaryFile(mode='w+') as tmpf:
+            tmpf.write("data_test")
+            tmpf.flush()
+            cif = CifData(file=tmpf.name)
             cif.store()
             calc.add_link_from(cif, "cif")
 
         calc.store()
         calc._set_state(calc_states.TOSUBMIT)
-        with SandboxFolder() as f:
-            calc._store_raw_input_folder(f.abspath)
+        with SandboxFolder() as fhandle:
+            calc._store_raw_input_folder(fhandle.abspath)
 
         fd = FolderData()
-        with open(fd._get_folder_pathsubfolder.get_abs_path(
-                calc._SCHED_OUTPUT_FILE), 'w') as f:
-            f.write("standard output")
-            f.flush()
+        with io.open(fd._get_folder_pathsubfolder.get_abs_path(
+                calc._SCHED_OUTPUT_FILE), 'w', encoding='utf8') as fhandle:
+            fhandle.write(u"standard output")
+            fhandle.flush()
```

Comment:
the `flush` here and below are pointless since the file will be closed afterwards and therefore flushed.

Reply:
flush() removed.